### PR TITLE
[JEP-3] - Adding the "Released As" field and "Ready for Release" status to Jenkins JIRA

### PR DESCRIPTION
**Pull request by oleg-nenashev**
It is a follow-up to the old thread: https://groups.google.com/forum/#!topic/jenkinsci-dev/wzc4VLplHvs

CC @daniel-beck @i386 @orrc @uhafner who participated in the original discussion

### CODE REVIEW

File: jep/3/README.adoc [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+For plugins this change will be opt-in in order to avoid impact on plugin maintainers.
+
+== Specification
+
+=== UPDATE: Modify the workflow in the JENKINS project
+
+. Add new “Fixed But Not Released” state (aka "Stage Release")
+** Instead of "In Progress" => "Resolved", contributors can optionally move integrated fixed into the "Fixed But Not Released" state
+. All current transitions in JENKINS workflow should be retained
+JIRA tickets can be moved to “Resolved” state from any existing state
```
Comment (uhafner) [inline jep/3/README.adoc]: This sentence is shown right after the word retained without punctuation.

File: jep/3/README.adoc [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+//
+//
+// Uncomment when this JEP status is set to Accepted, Rejected or Withdrawn.
+//| Resolution
+//| :bulb: Link to relevant post in the jenkinsci-dev@ mailing list archives :bulb:
+
+|===
+
+
+== Motivation
```
Comment (bitwiseman) [inline jep/3/README.adoc]: Please see https://github.com/jenkinsci/jep/blob/master/jep/1/README.adoc#required-sections
This section should be the "Abstract".  

Also, unless there's a strong reason to deviate you should retain the suggested order, with "Motivation" after the "Specification".

**Comment by oleg-nenashev** [comment]
@bitwiseman @uhafner I have addressed the comments

**Comment by recena** [comment]
I agree with this proposal but won't be worth if maintainers do not use it properly.

**Comment by uhafner** [comment]
I already approved the PR 👍

**Comment by bitwiseman** [comment]
@oleg-nenashev Approved as Draft.

**Comment by oleg-nenashev** [comment]
@recena Could you please comment in the mailing list? Any feedback and improvement proposals will be helpful

File: jep/3/README.adoc [hunk 3]
```diff
@@ -0,0 +1,191 @@
...
+
+| Title
+| Adding the "Released As" field and "Ready for Release" status to Jenkins JIRA
+
+| Sponsor
+| link:https://github.com/oleg-nenashev[@oleg-nenashev]
+
+| Status
+// Uncomment the appropriate line.
+| Draft :information_source:
```
Comment (daniel-beck) [inline jep/3/README.adoc]: This entry should be *Draft Not Submitted*, Draft is the line below.

CC @bitwiseman

**Pull request merged**